src/cs2_demo_downloader/match_scraper.py

```diff
@@ -209,6 +223,17 @@ def scrape_matches(tab, webauth):
                 "match_score": "",
             }
 
+        # ex
```

Comment:
Elaborate on this comment. "ex" being "example" isn't immediately obvious until you actually read the code, so then the comment is a bit useless.

Reply:
Or you could just remove it 🤘